src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -941,6 +933,23 @@ private function putProductToCart(ProductInterface $product, string $tokenValue,
         );
     }
 
+    private function changeQuantityOfFirstProductInCart(string $tokenValue, int $quantity = 1): void
```

Comment:
As we have almost 1000 lines in this context, these methods for operating on cart like adding products or changing quantity could be moved to the CartContext